docs/README.md

```diff
@@ -1,30 +1,162 @@
+      "seller-table"
+    ]
+  },
+  "seller-table": {
+    "children": [
+      "vtex.store-components:product-name",
+      "seller-simulate-shipping",
+      "seller-head",
+      "seller-body"
+    ]
+  },
+  "seller-head": {
+    "children": [
+      "seller-head-cell#Seller",
+      "seller-head-cell#Price",
+      "seller-head-cell#Shipping",
+      "seller-head-cell#PriceWithShipping",
+      "seller-head-cell#BuyButton"
+    ]
+  },
+  "seller-head-cell#Seller": {
+    "props": {
+      "title": "Seller"
+    }
+  },
+  "seller-head-cell#Price": {
+    "props": {
+      "title": "Price"
+    }
+  },
+  "seller-head-cell#Shipping": {
+    "props": {
+      "title": "Shipping"
+    }
+  },
+  "seller-head-cell#PriceWithShipping": {
+    "props": {
+      "title": "Price With Shipping"
+    }
+  },
+  "seller-head-cell#BuyButton": {
+    "props": {
+      "title": "Add To Cart"
+    }
+  },
+  "seller-body": {
+    "children": [
+      "seller-row"
+    ]
+  },
+  "seller-row": {
+    "children": [
+      "seller-name",
+      "seller-price",
+      "seller-shipping",
+      "seller-price-with-shipping",
+      "seller-add-to-cart"
+    ]
+  }
+}
+```
+# Props Configuration
+
+
+#### seller-table
+
+| Prop name              | Type      | Description                                                                                                                                                                                                                                     | Default value |
+| ---------------------- | --------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------- |
+| `limitShownShippingInformation`       | `Number`  | Max number of shipping options to be shown.                                                                                                                                                                             | `3`         |
+
+
+#### seller-add-to-cart
+
+| Prop name              | Type      | Description                                                                                                                                                                                                                                     | Default value |
+| ---------------------- | --------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------- |
+| `isOneClickBuy`          | `Boolean` | Whether the button button add the product to the cart and stay in the page or not                                                                                                                        | false         |
```

Comment:
```suggestion
### `seller-add-to-cart` props

| Prop name              | Type      | Description                                                                                                                                                                                                                                     | Default value |
| ---------------------- | --------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------- |
| `isOneClickBuy`          | `boolean` | Defines if users will keep navigating in the same page once the Buy button was clicked on (`true`) or if they will be redirected (`false`)   | `false`         |
```

Reply:
Which page will users be redirected for? (considering that the `false`  value was configured)

Reply:
The user will be redirected for the cart page, for him to follow the checkout flow. I'll commit your suggestion and add this information too!